apps/shell/tash_internal.h

```diff
@@ -91,6 +91,11 @@ extern int tash_sleep(int argc, char **args);
 #ifdef CONFIG_TASH_USLEEP
 extern int tash_usleep(int argc, char **args);
 #endif
+#ifdef CONFIG_PM
+#define TASH_PM_DOMAIN_NAME "TASH"
+#define PM_DRIVER "/dev/pm"
+extern int tash_pm_domain_id;
```

Comment:
Instead of using global variable, let's set it as a static variable and provide a new function to get or to set it.

Reply:
Done